packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -11,7 +11,7 @@ const BodyContainer = classed('div', 'flex flex-col w-full pb-6');
 
 const PageBodyContainer = classed(
   BodyContainer,
-  'm-auto w-full laptop:border-l laptop:border-theme-divider-tertiary',
+  'm-auto w-full laptop:border-l laptop:border-r laptop:border-theme-divider-tertiary',
```

Comment:
I think we can simplify to `border-x` here:

```suggestion
  'm-auto w-full laptop:border-x laptop:border-theme-divider-tertiary',
```

Reply:
Oh, didn't know about that. Thanks! 🚀